content/en/guides/models/track/runs/_index.md

```diff
@@ -348,26 +348,30 @@ https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
 ### Customize how runs are displayed
-This section shows how to customize how runs are displayed in your project's **Workspace** and **Runs** tab, which share the same display configuration.
+This section shows how to customize how runs are displayed in your project's Workspace and Runs table.
 
 {{% alert %}}
 A workspace is limited to displaying a maximum of 1000 runs, regardless of its configuration.
 {{% /alert %}}
 
+#### Add or remove columns
 
-To customize which columns are visible:
-1. In the project sidebar, navigate to the **Runs** tab.
+To customize which columns are visible in the Runs table or Workspace:
+1. In the project sidebar, navigate to the **Runs** tab or the **Workspace** tab.
```

Comment:
And is the Runs tab different from the Runs table? I think that they both have the same settings.